find_me_a_sandwich/features/favorites/friends_favorites.feature

```diff
@@ -0,0 +1,29 @@
+Feature: friends favorites
+  As a user
+  I want to find the favorite meals of my friends
+  In order to decide what to eat
+
+  Background:
+    Given I have a valid account
+    And I am logged in
+    And there are merchants for 60606
+
+  Scenario: Happy
```

Comment:
This is not very descriptive for a scenario.  There may be many happy, sad and bad paths for a given feature.